src/components/ToDoListButtons.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="flex flex-nowrap">
+    <div>
+      <button
+        class="rounded-lg sm:px-[40px] sm:py-[15px] px-[15px] py-[5px] bg-green-500 text-white font-bold"
+        style="background-color: #38cb89"
```

Comment:
Why you use style here? Please declare a color variable inside tailwind config and use that as a class